data.sql

```diff
@@ -40,4 +40,57 @@ UPDATE animals SET species_id =
     WHEN name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' THEN 4
     WHEN name = 'Angemon' OR name = 'Boarmon' THEN 5
   END;
-  
\ No newline at end of file
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specialties (vet_id, species) VALUES
+(1, 'Pokemon'),
+(3, 'Digimon'),
+(3, 'Pokemon'),
+(4, 'Digimon');
+
+-- Insert data for vets
+INSERT INTO vets (name, age, graduation_date)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+       ('Maisy Smith', 26, '2019-01-17'),
+       ('Stephanie Mendez', 64, '1981-05-04'),
+       ('Jack Harkness', 38, '2008-06-08');
```

Comment:
- [ ] Duplicated lines of code, Kindly always make sure your codebase is kept as simple and short as possible. These exact lines of code are the same as line `45` to `50`.